### PR TITLE
fix missing curly brace in js

### PR DESCRIPTION
**Pull request by chiazo**
also removed old login logic!

### CODE REVIEW

File: eisenhower-matrix/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,46 +1,63 @@
+// check login status on load
+document.addEventListener("DOMContentLoaded", function () {
+  // every time there is a submission, fetch tasks
+  getTasks();
+<<<<<<< HEAD
```
Comment (deeptivaidy) [inline eisenhower-matrix/src/main/webapp/script.js]: Can we get rid of this marker?

**Review by deeptivaidy — DISMISSED** [review]

File: eisenhower-matrix/src/main/webapp/script.js (hunk 1 above)
Comment (chiazo) [reply eisenhower-matrix/src/main/webapp/script.js]: oops! yes

**Comment by chiazo** [comment]
@deeptivaidy @deeptivaidy @garciajess can y'all rereview this? :)

File: eisenhower-matrix/src/main/webapp/script.js (hunk 1 above)
Comment (deeptivaidy) [reply eisenhower-matrix/src/main/webapp/script.js]: Sorry one last thing, the api key changed so can one of us update that real quick?

**Comment by chiazo** [comment]
@deeptivaidy the api key is updated.

**Comment by garciajess** [comment]
Thanks for the updates!

**Review by garciajess — APPROVED** [review]

**Pull request merged**